scalaj/client/src/main/scala/endpoints/scalaj/client/Urls.scala

```diff
@@ -28,9 +28,13 @@ trait Urls extends algebra.Urls {
   }
 
   class Url[A](val toReq: A => HttpRequest)
+  
+  implicit lazy val queryStringPartialInvFunctor: PartialInvariantFunctor[QueryString] = new PartialInvariantFunctor[QueryString] {
+    def xmapPartial[A, B](fa: QueryString[A], f: A => Option[B], g: B => A): QueryString[B] = fa compose g
+  }
 
   implicit lazy val queryStringParamPartialInvFunctor: PartialInvariantFunctor[QueryStringParam] = new PartialInvariantFunctor[QueryStringParam] {
-    def xmapPartial[A, B](fa: A => List[String], f: A => Option[B], g: B => A): B => List[String] = fa compose g
+    def xmapPartial[A, B](fa: QueryStringParam[A], f: A => Option[B], g: B => A): QueryStringParam[B] = fa compose g
```

Comment:
We noticed that `QueryStringParam` was not used in in `xmapPartial`

Reply:
FYI, both are correct since `QueryStringParam[A]` is an alias for `A => List[String]` in `scalaj.client.Urls`. That being said, I prefer using the `QueryStringParam` name because it reflects better the intent. Thanks for fixing this!